src/System.Management.Automation/engine/hostifaces/Command.cs

```diff
@@ -73,6 +73,12 @@ internal Command(string command, bool isScript, bool? useLocalScope)
             _useLocalScope = useLocalScope;
         }
 
+        internal Command(string script, string scriptPath, bool? useLocalScope)
+            : this(script, true, useLocalScope)
```

Comment:
```suggestion
            : this(script, isScript: true, useLocalScope)
```

Reply:
I didn't do this because of the convention used in this class - [the first constructor](https://github.com/PowerShell/PowerShell/blob/16ec1cc32f9e60c9d76b074c865c32a5d2acf5a2/src/System.Management.Automation/engine/hostifaces/Command.cs#L28) in the class also doesn't use named `isScript` parameter. Do you want to add this change anyway?

Reply:
Yes, it makes code more readable.

Reply:
Fixed with last commit together with other occurrences in the same class.